.docker/populate-services.sh

```diff
@@ -19,10 +19,10 @@ for service in "${content_services[@]}"; do
         id=$(basename "$(dirname "${file}")")
         version=$(basename -- "${file%.*}")
 
-        [[ "$(curl --verbose --silent --show-error --output /dev/null --request PUT "http://localhost:${HTTP_PORT_GATEWAY}/${service}/items/${id}/versions/${version}" --upload-file "${file}" --write-out '%{http_code}')" == "204" ]]
+        [[ "$(curl --verbose --silent --show-error --output /dev/null --request PUT --header "Host: unstable--api-gateway.libero.pub" "http://localhost:${HTTP_PORT}/${service}/items/${id}/versions/${version}" --upload-file "${file}" --write-out '%{http_code}')" == "204" ]]
```

Comment:
Can this be moved to the line above and do something like this?

```bash
curl --verbose \
       --silent \
       --show-error \
       --output /dev/null \
       --request PUT \
       --header "Host: unstable--api-gateway.libero.pub" \
       "http://localhost:${HTTP_PORT}/${service}/items/${id}/versions/${version}" \
       --upload-file "${file}" \
       --write-out '%{http_code}'
```

Reply:
Should be possible to reformat and extract a variable, yes